src/settings.cpp

```diff
@@ -1287,6 +1288,7 @@ static void HandleSettingDescs(IniFile &generic_ini, IniFile &private_ini, IniFi
 		proc_list(private_ini, "servers", _network_host_list);
 		proc_list(private_ini, "bans", _network_ban_list);
 		proc_list(private_ini, "server_authorized_keys", _settings_client.network.server_authorized_keys);
+		proc_list(private_ini, "rcon_authorized_keys", _settings_client.network.rcon_authorized_keys);
```

Comment:
Hmm .. with `rcon_authorized_keys`, I now wonder if we should also name it `client_authorized_keys`, instead of `server_`?

Reply:
I've chosen to use `s/password/authorized_keys/` to determine the setting names and not put any more thought than that into it. It'll keep the naming at least consistent within the configuration.

Reply:
I guess. Naming is still hard :P